components/Header.js

```diff
@@ -1,18 +1,11 @@
 import Contributors from './Contributors';
 import styles from './css/Header.module.css';
 
+
 const Header = (props) => {
 
   return (
     <React.Fragment>
```

Comment:
This `React.Fragment` isn't necessary anymore.

Reply:
removed in the latest pull request.